storage/clickhousespanstore/mocks/converter_test.go

```diff
@@ -0,0 +1,44 @@
+package mocks
+
+import (
+	"database/sql/driver"
+	"testing"
+	"time"
+
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConverterMock_ConvertValue(t *testing.T) {
+	converter := ConverterMock{}
+
+	type test struct {
+		valueToConvert interface{}
+		expectedResult driver.Value
+	}
+
+	testCases := map[string]test{
+		"string value":       {"some string value", driver.Value("some string value")},
```

Comment:
Please use names in struct options. It is recommended good practice.

Reply:
Can you please give me some guideline for writing table driven tests?

Reply:
The test looks good, but the struct definition should list param names.

Reply:
@pavolloffay done